back-office/shop-parameters/customer-settings/customers.md

```diff
@@ -0,0 +1,23 @@
+# **SPECIFICATIONS - CUSTOMERS SETTINGS > CUSTOMERS PAGE**
+
+
+## General
+
+_As a merchant, I want to be able to define my preferences to manage my customers._
+
+**Re-display cart at login.** By default, this option is disabled. When enabled, **if a customer has left a cart unchecked-out, it displays it again when he/she logs back in**. It should even memorize at what step of the checkout the customer stops so that he/she can go back directly to where the checkout was stopped. A tooltip is available: `After a customer logs in, you can recall and display the content of his/her last shopping cart.` in `Admin.Shopparameters.Help`.
+
+**Send an email after registration.** Enabled by default, the setting allows any newly-created customer to **receive by email a summary of his/her account information after registration**. It triggers the _account_ email, available for both email templates, _classic_ and _modern_, in the Design > Email Theme section of the back office. A tooltip is available: `Send an email with summary of the account information (email, password) after registration.` in `Admin.Shopparameters.Help`.
+
+**Password reset delay.** Merchants can **prevent customers from generating new passwords too often**. It limits the frequency at which a customer can generate a new password for his/her account. By default, it is set at 360 minutes (= 6 hours). A tooltip is available: `Minimum time required between two requests for a password reset.` in `Admin.Shopparameters.Help`.
```

Comment:
Actually, I'm not sure, I need to double-check with a locally installed store to specify the complete behavior - from the moment the error notification is displayed to the email with the reset link. It might not be able to do it quickly but I will add it as soon as possible, thanks!